jupyterhub/templates/hub/deployment.yaml

```diff
@@ -55,6 +55,8 @@ spec:
       {{- end }}
       securityContext:
         fsGroup: {{ .Values.hub.fsGid }}
+        # Don't allow any process to execute as root inside the container
+        allowPrivilegeEscalation: false
```

Comment:
Ah, this is within the pod, it needs to be within the container I think.

See: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod